docs/user/writing_documentation.md

```diff
@@ -0,0 +1,56 @@
+
+# Writing Documentation
+When adding new features to garage, it is important that these features are well-documented and accessible. Garage's high-level docs contain information on workflows and processes that are regularly used by developers and end-users. They are home to various how-to's and examples that help others use garage effectively.
+
+High-level documentation should focus on describing such workflows, or, in the case of newly-added features, describing how these features might be used when running experiments. For code-level documentation such as docstrings and style guides, please see the [CONTRIBUTING.md](https://github.com/rlworkgroup/garage/blob/master/CONTRIBUTING.md).
+
+
+## Where to Put Documentation
+You can choose to update the existing docs under the `docs/user` directory, or create a new page dedicated to the subject you are documenting.
+
+In certain cases, it may be beneficial to include your documentation in a README file in a directory specific to that new feature or workflow. Garage's `docker` folder, for example, contains the main Dockerfile and `entrypoint.sh` scripts that are used to run the docker builds. This folder contains a separate README with details on garage's docker images and how to run them.
```

Comment:
I think we've decided that actually these README's should just point to a docs page in `docs/user`.

Reply:
agreed -- we will be moving this to docs/ as well